mysql-test/suite/rocksdb_hotbackup/include/stream_run.sh

```diff
@@ -29,36 +32,42 @@ rm -rf $dest_data_dir/
 mkdir $dest_data_dir
 
 COPY_LOG="${MYSQL_TMP_DIR}/myrocks_hotbackup_copy_log"
+SIGNAL_CONDITION=""
+SIGNAL_FILE=${MYSQL_TMP_DIR}/myrocks_hotbackup_signal
```

Comment:
Probably just need to add a line to ensure the COPY_LOG and SIGNAL_FILE don't exist before the test starts. If there was anything left over from an old run, it could still result in a timing issue.